docs/introduction/abstract-filesystem.md

```diff
@@ -1,7 +1,7 @@
 # Abstract Filesystem
 One of the goals of the Shopsys Framework is to give you tools to make your e-commerce platform scalable.
 One of the requirements for scalable application is separated file storage that is accessible from all application instances.
-We use abstract filesystem - [Flysystem](https://github.com/thephpleague/flysystem) via [OneUpFlysystemBundle](https://github.com/1up-lab/OneupFlysystemBundle) to fulfill this requirement.
+We use abstract filesystem - [Flysystem](https://github.com/thephpleague/flysystem).
```

Comment:
i am not sure whether this should not be as part of html comments `<!--  blablabla change during release blablabla -->` that will be updated after new release is done but .... i think this should be solved during BV.